src/kernel/retype_tbl.c

```diff
@@ -79,6 +79,13 @@ mod_mem_type(void *pa, const mem_type_t type)
 	struct retype_info_glb *glb_retype_info;
 
 	assert(pa); 	/* cannot be NULL: kernel image takes that space */
+	/*do
```

Comment:
Retype table is done in a very naive way, that is to reduce the retyping granularity to 256Byte.